src/components/autocomplete/js/autocompleteDirective.js

```diff
@@ -263,8 +266,32 @@ angular
  * </md-autocomplete>
  * </hljs>
  *
+ * ### Specifying Repeat Mode
+ * You can use md-mode to specify whether to use standard or virtual lists for
+ * rendering autocomplete suggestions.
+ * The md-mode accepts two values:
+ * - `virtual` (default) Uses md-virtual-repeat to render list items. If the
```

Comment:
Looks like there is a dangling "If the".

Reply:
Got it. Thanks!